### DISCUSSION TITLE
How can a player have a higher combo than auto mod? (no 2b map)

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22837] [Q&A]
https://lazer.ppy.sh/beatmapsets/888109#osu/1966785

Found this, 3rd score (knowledgeking ; users/8022517) literally cant be replicated, is that an issue where the map has been changed/updated and a score somehow survived from when it had more hit objects or something?

**Comment by Axilotl17** [comment]
could be a lot of things, but sometimes scores of edited beatmaps still submit. this is a known issue and will be fixed eventually.
